### DISCUSSION TITLE
No puedo acceder al taller de github

### DISCUSSION THREAD

**Discussion by Gil987654321** [Launch-X-Latam/MisionFrontEnd#327] [Q&A]
Me muestra el siguiente mensaje: 
![image](https://user-images.githubusercontent.com/92352192/200073314-ab57a925-f45a-46be-b81b-f3ea46997000.png)

**Comment by DaveOval** [comment]
yo no me entere por que no me notifico nunca teams :(

**Reply by LJimena70** [reply]
Yo me registré y no me llegó nada :C

![Captura](https://user-images.githubusercontent.com/98842390/200097720-9e0ad57f-9312-4c93-8673-d09c49d76f65.JPG)
